### ISSUE TITLE
Transparency Bug with Armored Mobs, Weather Mod... 

### ISSUE THREAD

**Issue by YeraX7** [nvb-uy/Hysteria#1]
### Transparency Bug with Armored Mobs and Weather Mod in Minecraft Shader

**Description:**
I'm using a shader alongside my modpack, and I've noticed a bug where mobs equipped with armor appear transparent or translucent if the armor is enchanted. Additionally, elements from the Weather, Storms, and Tornadoes mod also appear transparent.

In an attempt to fix this issue, I tried disabling the "Colored Shadows" option in the shader's config. However, this action disables the shader entirely and an error message appears in the chat:

`composite.fsh: composite.fsh: 0(523) : error C1503: undefined variable "shadowcolor0"`

This error might be related to the transparency issue mentioned above.

**Additional Testing:**
Testing in a separate world with only the Weather, Storms, and Tornadoes mod and the shader, I found that the transparency issue persists with elements from the mod. However, the transparency issue with armored mobs seems to be resolved in this isolated environment.

**Visual Evidence:**

Thank you for your attention.

![2024-07-06_22 11 23](https://github.com/nvb-uy/Hysteria/assets/109879473/07642019-a14e-4185-9e41-66733005b157)
![ezgif-6-1aa7c348f3](https://github.com/nvb-uy/Hysteria/assets/109879473/35f5e3d7-7de7-41e4-940d-ba323c002cf6)

### Minecraft Version

1.20.1

### Which shader mod are you using?

Iris

### Shader mod version

1.1.0

### Shader settings

```conf
Default config in any profile
```

### Relevant log output

```text
composite.fsh: composite.fsh: 0(523) : error C1503: undefined variable "shadowcolor0"
```

**Comment by loti1234** [comment]
ngl this looks cool af

**Comment by YeraX7** [comment]
> ngl this looks cool af

Yeah, but when it's not enchanted or you're on your own, you basically look invisible hahaha. Also, all transparent particles, like rain or whatever you can think of, look very bad. I love the shader, and it's perfect for my apocalypse modpack and the server I'm creating, but it's a shame that it has this problem.

**Comment by loti1234** [comment]
oh

**Comment by Ulizky** [comment]
I'm having the same problem. I love the shader, and this little detail of not being able to see the armor is very annoying. I would really appreciate it if you fix this bug. Thanks in advance.

**Comment by JoeyDagger** [comment]
Found the issue, for my case at least
- Navigate to the config for Shoulder Surfing Reloaded (shouldersurfing-client.toml)
  - Set:
    - adjust_player_transparency = **false**

**Comment by S4Y3D-henti** [comment]
Same issue here and also my game goes black and light blue when I'm looking in certain directions. I'm unsure on why this happens.
